src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const STAT = 'bookStore/categories/STATUS';
+
+const initialState = 'status Unknown';
+
+export default function categoryReduce(state = initialState, action) {
+  switch (action.type) {
+    case STAT:
+      return 'still constructing';
```

Comment:
- [ ] According to the requirements of this project, the check status action should return " `Under construction` ", please kindly fix that.